int64 blue_dis
int64 blue_ang1
int64 blue_ang2
int64 blue_left
int64 blue_right
int64 blue_fix_x
int64 blue_fix_y
int64 blue_fix_angle
int64 blue_fix_distance


int64 yellow_dis
int64 yellow_ang1
int64 yellow_ang2
int64 yellow_left
int64 yellow_right
int64 yellow_fix_x
int64 yellow_fix_y
int64 yellow_fix_angle
int64 yellow_fix_distance